source/features/tags-dropdown.css

```diff
@@ -10,7 +10,19 @@
 	max-width: 300px;
 }
 
-/* Space out buttons on Releases page */
-.subnav > .float-right {
-	margin-left: 20px;
+/* Limit dropdown height */
+:root .rgh-tags-dropdown .SelectMenu-list {
+	max-height: 300px !important;
+}
```

Comment:
Avoids this:

<details>
<summary>long_tags.png</summary>

![image](https://user-images.githubusercontent.com/46634000/153719521-6077cfee-894c-43cd-a32f-bab42389b90f.png)

</details>

By the way, it looks like GitHub devs are even more confused about their own markup than we are :laughing: 

![image](https://user-images.githubusercontent.com/46634000/153719726-4f2d7d01-06c0-40a6-beda-562b67a00445.png)